doc/man3/CRYPTO_THREAD_run_once.pod

```diff
@@ -113,12 +113,13 @@ The other functions return 1 on success, or 0 on error.
 =head1 NOTES
 
 On Windows platforms the CRYPTO_THREAD_* types and functions in the
-openssl/crypto.h header are dependent on some of the types customarily
-made available by including windows.h. The application developer is
+F<openssl/crypto.h> header are dependent on some of the types customarily
```

Comment:
Elsewhere @levitte used `F<< <openssl/crypto.h> >>`. Could you please do the same here and in other places mentioning headers below.

Reply:
I had deliberately not done `F<< <` to avoid stepping on @levitte's PR.  Changed.